docker/auto-setup.sh

```diff
@@ -36,6 +36,11 @@ set -eux -o pipefail
 : "${POSTGRES_USER:=}"
 : "${POSTGRES_PWD:=}"
 
+: "${SQL_TLS_ENABLED:=false}"
+: "${SQL_CERT:=}"
+: "${SQL_CERT_KEY:=}"
+: "${SQL_CA:=}"
```

Comment:
I'm confused regarding the `SQL_CERT`, `SQL_CERT_KEY`, and `SQL_CA` variables' usage. Why are they used differently than `SQL_TLS_ENABLED`?